tests/core/tests/test_command_export.py

```diff
@@ -0,0 +1,34 @@
+from io import BytesIO, TextIOWrapper
```

Comment:
Since the commands are under `management/commands` perhaps the tests could also be in a subdirectory `management/commands` ?

Reply:
I think it is best practice to keep tests for the same app within its app directory. I checked the Django codebase, and the tests are not in the `management/commands` subdirectory.